### PR TITLE
Edit docs for loading & displaying images

### PR DESCRIPTION
**Pull request by nickmcintyre**
Addresses #6389

 Changes:
I edited the inline documentation for functions for loading and displaying images to bring them more in line with the style guide.

@davepagurek @dkessner @MsQCompSci @limzykenneth @Qianqianye @stalgiag @cgusb @photon-niko @KleoP I'd love to incorporate any feedback you may have!

#### PR Checklist

- [x] `npm run lint` passes
- [x] [Inline documentation] is included / updated

### CODE REVIEW

**Review by davepagurek — APPROVED** [review]
Looks good! Nice work on `image()` especially, that's a hairy function signature to annotate!

**Comment by limzykenneth** [comment]
Looks good to me too. @nickmcintyre Feel free to merge when you are ready.

**Comment by Qianqianye** [comment]
Thanks @nickmcintyre. We will merge this PR after @dkessner and @MsQCompSci approve the changes.

File: src/image/loading_displaying.js
Comment (dkessner) [inline src/image/loading_displaying.js:21]: Perhaps this could be:
"interprets the first parameter in one of several ways."
 1) Path to local file, 2) URL, 3) base64 raw image data.

File: src/image/loading_displaying.js
Comment (dkessner) [inline src/image/loading_displaying.js:189]: Is it clear to all potential users that "download" == "save to a local file"?  (Is this clarified somewhere else in the docs?)

**Review by dkessner — APPROVED** [review]
I suggested a couple of minor clarifications, but the changes look good overall.

File: src/image/loading_displaying.js
Comment (nickmcintyre) [reply src/image/loading_displaying.js]: Good suggestion–fixed!

File: src/image/loading_displaying.js
Comment (nickmcintyre) [reply src/image/loading_displaying.js]: Good clarification–added!

**Pull request merged**
